### PR TITLE
Cleanup of automotive scanner debug output

### PR DESCRIPTION
**Pull request by polybassa**
…  and set ISOTPSoftSocket to blocking socket

### CODE REVIEW

File: scapy/contrib/automotive/uds_scan.py [hunk 1]
```diff
@@ -224,8 +223,9 @@ def cleanup(_, configuration):
         try:
             configuration["tps"].stop()
             configuration["tps"] = None
-        except (AttributeError, KeyError) as e:
-            log_automotive.debug("Cleanup TP-Sender Error: %s", e)
+        except (AttributeError, KeyError):
+            pass
+            # log_automotive.debug("Cleanup TP-Sender Error: %s", e)
```
Comment (gpotter2) [inline scapy/contrib/automotive/uds_scan.py:228]: Is this intended?

**Comment by gpotter2** [comment]
Hi, thanks for the pr! Sorry there are conflicts :/

File: scapy/contrib/automotive/uds_scan.py (hunk 1 above)
Comment (polybassa) [reply scapy/contrib/automotive/uds_scan.py:228]: Yes, this is intended. I wanted to cleanup the logging from this message.

**Comment by polybassa** [comment]
@gpotter2 Conflict resolved.

**Pull request merged**